### DISCUSSION TITLE
No tienes acceso a Google Sites

### DISCUSSION THREAD

**Discussion by ulises-ben-suhr** [dds-jv/foro#409] [Consultas generales] [answered]
Buenas noches!
Espero se encuentren bien, mi nombre es Ulises
Estuve revisando el sitio de la materia y pude acceder a todo el material de lectura (al menos todos los que intenté), pero cuando intento abrir la mayoría de los Ejercicios Cortos me sale: "No tienes acceso a Google Sites"
Algunos pude abrir, como el de Macowins(Parte 1 y 2) de la Selección de Comportamiento de la guía 3 y el de Aterrizar.com de Estado como Objeto, también de la guía 3
Estoy con el mail institucional, no se que puede ser, necesito estar en algún grupo de trabajo o tener un permiso especial?
Desde ya muchas gracias!!

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola, @ulises-ben-suhr!

Gracias por avisar. Estaremos corrigiendo los enlaces a la brevedad. 

¡Saludos!
